### PR TITLE
Fix/lint md files

### PR DESCRIPTION
**Pull request by scinos**
#### Changes proposed in this Pull Request

* Apply eslint fixes to MD files
* Configure exceptions for code blocks inside MD files 

#### Testing instructions

* N/A

### CODE REVIEW

File: packages/data-stores/README.md [hunk 1]
```diff
@@ -13,22 +13,19 @@ import React from 'react';
...
+	const verticals = useSelect( ( select ) => select( VERTICALS_STORE ).getVerticals() );
+
+	return (
+		<ul>
+			{ verticals.map( ( vertical ) => (
+				<li key={ vertical.vertical_id }>{ vertical.vertical_name }</li>
+			) ) }
+		</ul>
+	);
+};
```
Comment (tyxla) [inline packages/data-stores/README.md:28]: Would you like to add `export` the `VerticalSelect` component or add it to a `render()` method to make ESLint happy?

File: packages/data-stores/README.md [hunk 2]
```diff
@@ -42,7 +39,9 @@ When an application needs to use a store that should be configured, it may be he
 ```ts
 // vertical-store.ts
 import { DomainSuggestions } from '@automattic/data-stores';
```
Comment (tyxla) [inline packages/data-stores/README.md:41]: Seems like we can remove this import completely as it's not used at all.

**Review by tyxla — APPROVED** [review]
Nice one 👍  LGTM 🚢 

I've added a couple of non-blocking suggestions which you could address to please ESLint if you wish.

File: packages/data-stores/README.md (hunk 1 above)
Comment (scinos) [reply packages/data-stores/README.md:28]: Fixed, thanks!

File: packages/data-stores/README.md (hunk 2 above)
Comment (scinos) [reply packages/data-stores/README.md:41]: Fixed, thanks!

**Pull request merged**